playground/frontend/playground_components/tools/extract_symbols_java/src/main/java/com/playground/extract_symbols/ClassInfo.java

```diff
@@ -0,0 +1,9 @@
+package com.playground.extract_symbols;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ClassInfo {
+    Set<String> publicMethods = new HashSet<>();
+    Set<String> publicFields = new HashSet<>();
```

Comment:
`final`?

Reply:
In java `final` acts the same way as `const` in dart